src/Http/Wolverine.Http/CodeGen/QueryStringHandling.cs

```diff
@@ -38,16 +38,19 @@ public override void GenerateCode(GeneratedMethod method, ISourceWriter writer)
         var alias = Variable.VariableType.FullNameInCode();
         writer.Write($"{alias} {Variable.Usage} = default;");
 
+
         if (Variable.VariableType.IsEnum)
         {
             writer.Write($"{alias}.TryParse<{alias}>(httpContext.Request.Query[\"{Variable.Usage}\"], out {Variable.Usage});");
         }
-        else
+        else if (Variable.VariableType.IsBoolean())
```

Comment:
Not particularly happy about this code, but also not sure its worth the extra abstraction of maintaining a list of cultured types.  Thoughts?

Reply:
YAGNI for now is my call